packages/shared/src/components/layout/MainLayoutHeader.tsx

```diff
@@ -103,7 +103,7 @@ function MainLayoutHeader({
       <SearchPanel
         className={{
           container: classNames(
-            'left-0 top-0 z-header mx-auto bg-background-default py-3 tablet:left-16 laptop:left-0 laptop:bg-transparent',
```

Comment:
Just to make sure this does not affect when theme is turned off. I think it was initially added to make sure they have a background.

Reply:
I did check this with the theme turned off on mobile/tablet on the following pages:
1. search page - the search panel is shown on the top of the page
2. feed page - the search panel is not shown at all

On the search page, we have a bug at the moment, which hides the AI chat button below the search box. This has nothing to do with the starwars theme changes though. If we want to tackle that, I'd rather do it in a separate PR.
<img width="1013" alt="Screenshot 2024-05-02 at 11 12 51" src="https://github.com/dailydotdev/apps/assets/9974711/089f8565-d09c-406f-b925-e43340581076">

Reply:
If u have a bit of time, do u mind raising a PR for that @kopancek ?

Reply:
For the sake of transparency, here's the issue in sprint added for it: https://dailydotdev.atlassian.net/browse/MI-346